src/scenes/CasAuthenticated/CasAuthenticated.js

```diff
@@ -12,23 +12,22 @@ const Successful = styled.div`
 `;
 
 const CasAuthenticated = () => {
-  const { state } = useContext(UserContext);
+  const { state } = useContext(BackendContext);
 
-  let organisaatio = undefined;
   let ytunnus = undefined;
-  if (state.oppilaitos && state.oppilaitos.organisaatio) {
-    organisaatio = state.oppilaitos.organisaatio;
-    if (organisaatio) {
-      if (organisaatio.ytunnus) {
-        ytunnus = organisaatio.ytunnus;
-      }
-    }
+  if (state.organisaatio) {
```

Comment:
These kinds of implementations can use ramda path instead.
One liner: `const ytunnus = R.path(["organisaatio", "raw", "ytunnus"], state);`

Reply:
Which is my change, but is this really better that way?

Reply:
Will be changed, to make sure raw is existing.